ui-lib/src/main/java/cash/z/ecc/ui/screen/profile/view/ProfileView.kt

```diff
@@ -108,7 +108,8 @@ private fun ProfileContents(
     onAddressBook: () -> Unit,
     onSettings: () -> Unit,
     onCoinholderVote: () -> Unit,
-    onSupport: () -> Unit
+    onSupport: () -> Unit,
+    isHidden: Boolean = false
```

Comment:
I'd like to suggest a more descriptive variable name, to make the public API more clear about what will be hidden.

```suggestion
    isAddressBookEnabled: Boolean = false
```

Reply:
Thanks a lot for the suggestion. I have changed the var name.

I have also addressed the failed connectedDevice test by adding `@ignore` annotation to the relevant test.

Cheers.